src/track.ts

```diff
@@ -80,7 +80,15 @@ const getBaseRequestBody = (
         ? Math.floor(client.timestamp / 1e3)
         : client.timestamp,
     event_source_url: payload.dl || client.url.href,
-    data_processing_options: [],
+    ...(settings.dataProcessingOptions && {
+      data_processing_options: ['LDU'],
+      ...(settings.dataProcessingOptionsCountry && {
+        data_processing_options_country: settings.dataProcessingOptionsCountry,
```

Comment:
Things generally arrive as strings, and so will be sent as strings too. The FB docs are saying they're expecting an integer. Have we checked that it works even when we're sending a string?

Reply:
We can test that the request actually makes it to FB. But I think we can't tell if the request is processed differently by Facebook or not. At least they do not mention what actually should happen and if it visible in the UI. Perhaps the best thing would be testing it with the real CF account?

Reply:
I think the best thing would be to send an integer, e.g.

```
...(settings.dataProcessingOptionsCountry && {
        data_processing_options_country: parseInt(settings.dataProcessingOptionsCountry),
```

Reply:
Oh okay! That's easy :)  Making the changes.